fastchat/llm_judge/gen_model_answer.py

```diff
@@ -100,7 +108,7 @@ def get_model_answers(
         choices = []
         for i in range(num_choices):
             torch.manual_seed(i)
-            conv = get_conversation_template(model_id)
+            conv = get_conversation_template(model_path)
```

Comment:
This is needed because the `model_id` is often not identical to the model path and it conflicts with `is_adapter_model()` which tries to look up the string on the Hub